test/unit/extractor/vue/tComponent.test.ts

```diff
@@ -22,7 +22,7 @@ describe('<T>', () => {
     const expected = [{ keyName: 'key1', line: 3 }];
 
     const code = `
-      <template> <!-- @tolgee/vue -->
+      <template>
```

Comment:
Nitpick: I would appreciate those cleanups in a separate commit, but probably just a matter of taste.